src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -117,10 +117,17 @@ public void setPerson(Person target, Person editedPerson) {
     }
 
     @Override
-    public void setPerson(Person target, Person editedPerson, Order order) {
+    public void setPersonAdd(Person target, Person editedPerson, Order order) {
```

Comment:
Perhaps, you could name this better, such as:
1. updatePersonAndAddOrder
2. setPersonAndAddOrder
3. updatePersonAndOrder
~~4. updatePersonOrderData~~ removed because its ambiguous 
5. refreshPersonAddOrder

Reply:
Changed to `setPersonAndAddOrder`